app/models/contact_email.rb

```diff
@@ -4,6 +4,4 @@
 # Models an email in the database
 class ContactEmail < ApplicationRecord
   belongs_to :emailable, polymorphic: true
-
-  validates :email, format: { with: Devise.email_regexp }, allow_blank: true
```

Comment:
need to remove model level validation for `ContactEmail`, else you cannot save a draft collection with invalid emails (since it needs to persist to the database, and the model level validation will block it)